docs/build.sbt

```diff
@@ -3,7 +3,7 @@ val ScalaVersion = "2.12.8"
 val AkkaVersion: String   = sys.props.getOrElse("lagom.build.akka.version", "2.5.22")
 val JUnitVersion          = "4.12"
 val JUnitInterfaceVersion = "0.11"
-val ScalaTestVersion      = "3.0.5"
+val ScalaTest             = "3.0.8-RC2"
```

Comment:
The convention in this build for version `val`s is `FooVersion`.
```suggestion
val ScalaTestVersion      = "3.0.8-RC2"
```

Reply:
Rebase mistake